src/components/DappsGallery/content/DappsGalleryContent.js

```diff
@@ -0,0 +1,23 @@
+import First from '../../../../static/img/tools/universalpage_logo.png';
+import Second from '../../../../static/img/tools/stakingverse_logo.jpeg';
+import Third from '../../../../static/img/tools/universalpage_logo.png';
+import Fourth from '../../../../static/img/tools/stakingverse_logo.jpeg';
+
+export const dappsContent = [
+  {
+    project: First,
+    link: 'https://partybid.app',
+  },
+  {
+    project: Second,
+    link: 'https://gallery.fwb.help/',
+  },
+  {
+    project: Third,
+    link: 'https://mirror.xyz/',
+  },
+  {
+    project: Fourth,
+    link: 'https://catalog.works/',
+  },
+];
```

Comment:
There is no need for an extra file at this stage, you can just put this in the component file it will make it easier to read